src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -46,16 +47,31 @@ export default function FieldTemplate(props) {
   let errorSpanId;
   let errorSpan;
   let errorClass;
+  let errorStyle = 'none';
   if (hasErrors) {
     errorClass = isDateField ? 'input-error-date' : 'usa-input-error';
     errorSpanId = `${id}-error-message`;
+    errorStyle = 'block';
     errorSpan = (
       <span className="usa-input-error-message" role="alert" id={errorSpanId}>
         <span className="sr-only">Error</span> {rawErrors[0]}
       </span>
     );
   }
 
+  if (!environment.isProduction()) {
```

Comment:
I think this should be in the `if (hasErrors) {`

Reply:
that block should always be present, the part that is toggling off the div is the ```errorStyle``` which is inside the ```if (hasError) {``` block.

I'll take a look at the unit test.